src/Cynthia.Card/src/Cynthia.Card.Server/Locales/pl.json

```diff
@@ -3083,7 +3083,7 @@
           },
           "70032": {
               "Name": "Gascon",
-              "Info": "Rozmieszczenie: Przenieś wszystkie wrogie jednostki do losowych rzędów. Za każdą przeniesioną jednostkę otrzymaj 2 pkt obrażeń. Wzmocnij się o 1 za każdym razem gdy jednostka zostanie poruszona w trakcie Twojej tury.",
```

Comment:
This remove is not intended, is it?

Reply:
The removal was intended but I forgot to put back the new description, thanks for noticing, I fixed it.